views/js/qtiCreator/widgets/interactions/graphicGapMatchInteraction/states/Correct.js

```diff
@@ -92,6 +91,7 @@ define([
         instructionMgr.removeInstructions(interaction);
 
         //initialize again the widget's paper
+        interaction.paper = null;
```

Comment:
**question:** What is the purpose of this? The property is assigned twice. Is it for working around some memory leaks or to manage the reflow? AFAIK, the `interaction` is not a DOM element...

Reply:
Issue when switch from Response to Question after choice exist
![image](https://user-images.githubusercontent.com/25976342/214315085-cc1ddfbd-3dc7-4299-8750-62b35d3a7acd.png)
Because on `createPaper` will be call `resize`, that has callback `scaleGapList` and we have condition `if(this.element.paper)` - but we don't have elements because `createPaper` is not finished

https://github.com/oat-sa/extension-tao-itemqti/blob/092e6fa5462d2d3cc44386637fa7c9eeba704e6f/views/js/qtiCreator/widgets/interactions/graphicGapMatchInteraction/Widget.js#L90-L116